mobile/app/App_Resources/Android/src/main/res/values/strings.xml

```diff
@@ -153,9 +154,9 @@
   <string name="_score_header_attributes_1IpA9K">"Attributs (%1$d%%)"</string>
   <string name="_score_done_1O5zTM">"Tu as complété les %1$d %2$s. %3$s"</string>
   <string name="_score_improve_mDxz3">"Tu as fait %1$d sur %2$d %3$s. %4$s"</string>
-  <string name="_score_desc_1O5xmK">"Ton degré d\'individualité est une évaluation faite par le système que tu es bien un individu unique. Plus ce degré est élevé, plus le système sera confiant que tu ne va pas l\'attaquer en te faisant passer pour plusieurs personnes."</string>
-  <string name="_score_header_W8cDM">"Degré d\'individualité"</string>
-  <string name="_score_title_wH00B">"Individualité"</string>
+  <string name="_score_desc_1O5xmK">"L\'accomplissement montre combien des functionlaités de l\'app tu as déjà utilisé. Plus tu utilises les différentes functionalités, plus ton accomplissement sera élevé."</string>
```

Comment:
```suggestion
  <string name="_score_desc_1O5xmK">"L\'accomplissement montre combien des functionnalités de l\'app tu as déjà utilisé. Plus tu utilises les différentes functionalités, plus ton accomplissement sera élevé."</string>
```

Reply:
Verify the verifier: Il y a encore un 'o' dans fonctionnalité...

Reply:
Good catch; edited